guides/common/modules/ref_webhooks-available-events.adoc

```diff
@@ -0,0 +1,45 @@
+[id="webhooks-available-events_{context}"]
+= Available Events
+
+The following are the webhook events available from the {ProjectWebUI}.
+
+[cols="50%,50%",options="header"]
+|====
+|Event name |Description
+|Actions Katello Content View Promote Succeeded |A content view was successfully promoted.
+|Actions Katello Content View Publish Succeeded |A repository was successfully synchronized.
+|Actions Remote Execution Run Host Job Succeeded |A generic remote execution job succeeded for a host. This event is emitted for all Remote Execution jobs, when complete
+|Actions Remote Execution Run Host Job Katello Errata Install Succeeded |Install errata using the Katello interface
+|Actions Remote Execution Run Host Job Katello Group Install Succeeded |Install package group using the Katello interface
+|Actions Remote Execution Run Host Job Katello Package Install Succeeded |Install package using the Katello interface
+|Actions Remote Execution Run Host Job Katello Group Remove |Remove package group using the Katello interface
+|Actions Remote Execution Run Host Job Katello Package Remove Succeeded |Remove package using the Katello interface
+|Actions Remote Execution Run Host Job Katello Service Restart Succeeded |Restart Services using the Katello interface
+|Actions Remote Execution Run Host Job Katello Group Update Succeeded |Update package group using the Katello interface
+|Actions Remote Execution Run Host Job Katello Package Update Succeeded |Update package using the Katello interface
+|Actions Remote Execution Run Host Job Foreman OpenSCAP Run Scans Succeeded |Run OpenSCAP scan
+|Actions Remote Execution Run Host Job Ansible Run Host Succeeded |Runs an Ansible playbook containing all the roles defined for a host
+|Actions Remote Execution Run Host Job Ansible Run Capsule Upgrade Succeeded |Upgrade Capsules on given Capsule server hosts
+|Actions Remote Execution Run Host Job Ansible Configure Cloud Connector Succeeded |Configure Cloud Connector on given hosts
+ifdef::satellite[]
+|Actions Remote Execution Run Host Job Ansible Run Insights Plan Succeeded |Runs a given maintenance plan from Red Hat Access Insights given an ID.
+endif::[]
+|Actions Remote Execution Run Host Job Ansible Run Playbook Succeeded |Run an Ansible playbook against given hosts
+|Actions Remote Execution Run Host Job Ansible Enable Web Console Succeeded |Run an Ansible playbook to enable the web console on given hosts
+|Actions Remote Execution Run Host Job Puppet Run Host Succeeded |Perform a single Puppet run
+|Actions Remote Execution Run Host Job Katello Module Stream Action Succeeded |Perform a module stream action using the Katello interface
+|Actions Remote Execution Run Host Job Leapp Pre-upgrade Succeeded |Upgradeability check for RHEL 7 host
+|Actions Remote Execution Run Host Job Leapp Remediation Plan Succeeded |Run Remediation plan with Leapp
+|Actions Remote Execution Run Host Job Leapp Upgrade Succeeded |Run Leapp upgrade job for RHEL 7 host
```

Comment:
All those `Action` events will trigger webhooks only on `success`, so if an action failed, a webhook is not triggered.